.changeset/rare-crabs-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/core-components': minor
+---
+
+Parse unicode characters in name for avatar component
```

Comment:
Hmm, I think you might have to add the word `unicode` to `.github/vale/Vocab/Backstage/accept.txt`